gradle.properties

```diff
@@ -1,5 +1,5 @@
 # modify this file to change project properties
-teamA=examplefuncsplayer
```

Comment:
I don't remember why I changed this file in this branch. I probably started working on the playtesting a little bit before I created a new branch.